arch/arm/core/cortex_r/Kconfig

```diff
@@ -86,6 +86,12 @@ config GEN_ISR_TABLES
 config GEN_IRQ_VECTOR_TABLE
 	default n
 
+config DISABLE_TCM_ECC
+	bool
+	default n
```

Comment:
This option is introduced here, right?
Any reason this is hidden?
Also, default n is not required for boolean Kconfigs